configure.ac

```diff
@@ -1446,6 +1446,10 @@ if test "$use_external_signer" != "no"; then
     ;;
     *)
       AC_MSG_CHECKING([whether Boost.Process can be used])
+      TEMP_CXXFLAGS="$CXXFLAGS"
+      dnl Boost 1.78 requires the following workaround.
+      dnl See: https://github.com/boostorg/process/issues/235
+      CXXFLAGS="$CXXFLAGS -Wno-error=narrowing"
```

Comment:
@hebasto I have to keep this flag in order to build, not sure what is the correct approach is, or if we only want to do it when boost=1.78.

Reply:
The reverse `CXXFLAGS="$TEMP_CXXFLAGS"` is required after the test.

Reply:
> The reverse `CXXFLAGS="$TEMP_CXXFLAGS"` is required after the test.

I mean, https://github.com/hebasto/bitcoin/commits/220222-boost-m

> But then it won't compile src/util/system.cpp, gives the same error.

What configuration with?

Reply:
> But then it won't compile src/util/system.cpp, gives the same error.

Then something like https://github.com/hebasto/bitcoin/commits/220222-boost-m2? It keeps the `-Wno-error=narrowing` only when it is necessary.

Reply:
Ok, I'll try that.

Reply:
Pushed an alternative that only ignores the error locally.